workshop1/slides/src/default.nix

```diff
@@ -0,0 +1,97 @@
+{ stdenv
+, fantasque-sans-mono
+, fira-code
+, fontconfig
+, latinmodern-math
+, lmodern
+, makeFontsConf
+, montserrat
+, norwester-font
+, pandoc
+, texlive
+, writeTextFile
+, ncurses
+, inotify-tools
+}:
+
+stdenv.mkDerivation {
+  name = "slides-maker";
+
+  buildInputs = [
+    pandoc
+    fontconfig
+    ncurses
+    (texlive.combine {
+      inherit (texlive)
+      scheme-small
+      #beamer
```

Comment:
Dead code

Reply:
This comment has an outdated label, but I can't figure out how to get the current status with github :<